### PR TITLE
added KubeCon NA 24 blog post

### PR DESCRIPTION
**Pull request by thtmnisamnstr**
Added a roundup blog post for KubeCon NA 2024.

### CODE REVIEW

**Comment by thtmnisamnstr** [comment]
http://www-testing-pulumi-docs-origin-pr-13298-79ad64af.s3-website.us-west-2.amazonaws.com/blog/kubecon-na-2024-roundup/

File: content/blog/kubecon-na-2024-roundup/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+meta_image: meta.png
+authors:
+    - gavin-johnson
+tags:
+    - kubecon
+    - kubernetes
+    - conferences
+---
+
+Pulumi is excited to be at KubeCon North America this week, the premier event for all things Kubernetes and cloud-native. Kubecon is the gathering place for developers, enterprises, and cloud native experts to meet and further the education and advancement of Kubernetes and cloud native computing. At Pulumi, we have a strong commitment to Kubernetes and continue to support the ecosystem with infrastructure management solutions that empower teams to automate, secure, and manage Kubernetes at scale.
```
Comment (isaac-pulumi) [inline content/blog/kubecon-na-2024-roundup/index.md:16]:
```suggestion
Pulumi is excited to be at KubeCon North America this week, the premier event for all things Kubernetes and cloud-native. KubeCon is the gathering place for developers, enterprises, and cloud native experts to meet and further the education and advancement of Kubernetes and cloud native computing. At Pulumi, we have a strong commitment to Kubernetes and continue to support the ecosystem with infrastructure management solutions that empower teams to automate, secure, and manage Kubernetes at scale.
```

File: content/blog/kubecon-na-2024-roundup/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+In this post, we’ll share some of the updates and improvements we've rolled out recently to better serve Kubernetes users.
```
Comment (isaac-pulumi) [inline content/blog/kubecon-na-2024-roundup/index.md:18]:
```suggestion
We've launched a slate of new and improved capabilities to serve Kubernetes users and to make it easier than ever to deploy, secure, manage and scale clusters and workloads. This article summarizes our recent investments. Got questions? Stop by the Pulumi KubeCon booth (R1) this week to chat with experts on the team. If you can't make it to  [join our workshop next week](https://www.pulumi.com/resources/pulumi-kubernetes-better-together/) to see how easy it is to manage Kubernetes with Pulumi. 
```

**Comment by thtmnisamnstr** [comment]
http://www-testing-pulumi-docs-origin-pr-13298-25eeaeb4.s3-website.us-west-2.amazonaws.com/blog/kubecon-na-2024-roundup/

**Comment by thtmnisamnstr** [comment]
http://www-testing-pulumi-docs-origin-pr-13298-31ccd7e3.s3-website.us-west-2.amazonaws.com/blog/kubecon-na-2024-roundup/

File: content/blog/kubecon-na-2024-roundup/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    - gavin-johnson
+tags:
+    - kubecon
+    - kubernetes
+    - conferences
+---
+
+Pulumi is excited to be at KubeCon North America this week, the premier event for all things Kubernetes and cloud-native (come visit us at booth R1). KubeCon is the gathering place for developers, enterprises, and cloud native experts to meet and further the education and advancement of Kubernetes and cloud native computing. At Pulumi, we are strongly committed to Kubernetes and continue to support the ecosystem with infrastructure management solutions that empower teams to automate, secure, and manage Kubernetes at scale.
+
+We've launched several new and improved capabilities recently to serve Kubernetes users and to make it easier than ever to automate, secure, manage, and scale clusters and workloads.
```
Comment (isaac-pulumi) [inline content/blog/kubecon-na-2024-roundup/index.md:18]:
```suggestion
We've launched several new and improved capabilities recently to serve Kubernetes users and to make it easier than ever to automate, secure, manage, and scale clusters and workloads. Stop by the Pulumi booth at KubeCon for a demo or join our [workshop](https://www.pulumi.com/resources/pulumi-kubernetes-better-together/) next week for an overview of these features: 
```

**Review by isaac-pulumi — APPROVED** [review]
Approving with a change

**Comment by thtmnisamnstr** [comment]
http://www-testing-pulumi-docs-origin-pr-13298-6a16637e.s3-website.us-west-2.amazonaws.com/blog/kubecon-na-2024-roundup/

**Pull request merged**